.github/workflows/k8s-content-pr-test.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: Gate / Kubernetes Test Content Parsing
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  datastream-parsing:
+    name: XCCDF Datastream parsing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Copy XCCDF files from existing content image
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 15
+          max_attempts: 3
+          retry_wait_seconds: 120
+          retry_on: error
+          command: |
+            mkdir -p content
+            docker pull ghcr.io/complianceascode/k8scontent:${{ github.event.number }}
+            docker run --rm -v $PWD/content:/content:z ghcr.io/complianceascode/k8scontent:${{ github.event.number }} bash -c "cp *.xml /content"
+      - name: Clone compliance operator repository
+        run: |
+          git clone https://github.com/ComplianceAsCode/compliance-operator.git
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '>=1.19.0'
+      - run: go version
+      - name: Install ginkgo
+        run: |
+          go install github.com/onsi/ginkgo/ginkgo@latest
+      - name: Run ginkgo tests and check if each XCCDF file is parsed correctly
+        run: |
+          export CONTENT_FOLDER=$PWD/content
+          cd compliance-operator
+          for file in $CONTENT_FOLDER/*.xml; do
+            export DEFAULT_CONTENT_DS_FILE_PATH=$file
+            ginkgo --focus "Testing for correct content parsing" ./pkg/utils | tee /tmp/ginkgo_output
+            if grep -q "0 Passed" /tmp/ginkgo_output && grep -q "0 Failed" /tmp/ginkgo_output; then
+              echo "XCCDF file $file is not parsed correctly"
+              exit 1
+            elif grep -q "SUCCESS!" /tmp/ginkgo_output; then
```

Comment:
Are we able to use return code to detect the outcome of the test? My concern with using grep is that it's probably more likely that the output would change. Using return codes is a more stable interface.